tests/test_krylov.py

```diff
@@ -0,0 +1,176 @@
+"""Tests comparing results of krylov pymadl function with apdl macro"""
+import os
+
+import numpy as np
+
+PATH = os.path.dirname(os.path.abspath(__file__))
+
+# Krylov Apdl Macro Files
+ulib_path = os.path.join(PATH, "test_files")
+kry_gensub = os.path.join(ulib_path, "KRYGENSUB.mac")
+kry_solve = os.path.join(ulib_path, "KRYSOLVE.mac")
+kry_expand = os.path.join(ulib_path, "KRYEXPAND.mac")
+
+
+def model_setup(mapdl, damp=0):
+    mapdl.clear()
+    mapdl.prep7()
+    mapdl.et(1, 185)
+    length = 1.5
+    width = 0.3
+    mapdl.block(0, length, -width / 2, width / 2, -width / 2, width / 2)
+    mapdl.vsweep(1)
+
+    # Define a material (nominal steel in SI)
+    mapdl.mp("EX", 1, 210e9)  # Elastic moduli in Pa (kg/(m*s**2))
+    mapdl.mp("DENS", 1, 7800)  # Density in kg/m3
+    mapdl.mp("NUXY", 1, 0.3)  # Poisson's Ratio
+
+    if damp == 1:
+        mapdl.mp("ALPD", 1, 0.001)
+
+    # Create BC
+    mapdl.nsel("S", "LOC", "X", 0)
+    mapdl.d("ALL", "UX")
+    mapdl.d("ALL", "UY")
+    mapdl.d("ALL", "UZ")
+
+    mapdl.allsel()
+
+    return length, width
+
+
+def solu_krylov(mapdl, frq_):
+    mapdl.allsel()
+    mapdl.run("/SOLU")
+    mapdl.antype("HARMIC")  # HARMONIC ANALYSIS
+    mapdl.hropt("KRYLOV")
+    mapdl.eqslv("SPARSE")
+    mapdl.harfrq(frq_)  # Set beginning and ending frequency
+    mapdl.nsubst(1)  # Set the number of frequency increments
+    mapdl.wrfull(1)  # GENERATE .FULL FILE AND STOP
+    output = mapdl.solve()
+    mapdl.finish()
+
+
+def test_krylov_with_point_load(mapdl):
+    # Case1 : Run Krylov Pymapdl
+    length, width = model_setup(mapdl, damp=1)
+    mm = mapdl.math
+    mapdl.jobname = "point_load_py"
+
+    # Parameters set for Krylov
+    maxQ = 10
+    frq_ = 100
+    Freqval_ = frq_
```

Comment:
Just use ``frq_`` or ``Freqval_``. Also you should not use underscore for parameter names.

```suggestion
    Freqval_ = frq_
```